mandible/log.py

```diff
@@ -1,9 +1,66 @@
+import json
 import logging
 import os
 from contextlib import contextmanager
+from functools import wraps
 from typing import Type
 
 
+class JSONFormatter(logging.Formatter):
+    def format(self, record):
+        log_record = {
+            "message": record.getMessage(),
+            "level": record.levelname,
+            "time": self.formatTime(record, self.datefmt),
+            "function_name": getattr(record, "function_name", None),
+            "memory_limit_in_mb": getattr(record, "memory_limit_in_mb", None),
+            "invoked_function_arn": getattr(record, "invoked_function_arn", None),
+            "aws_request_id": getattr(record, "aws_request_id", None),
+            "log_group_name": getattr(record, "log_group_name", None),
+            "log_stream_name": getattr(record, "log_stream_name", None),
+            "cumulus_version": getattr(record, "cumulus_version", None),
+            "step_function_name": getattr(record, "step_function_name", None),
+            "granule_name": getattr(record, "granule_name", None),
+            "core_version": getattr(record, "core_version", None),
```

Comment:
I think this should be `cirrus_core_version`. The reasoning is that the Cumulus does refer to a portion of the Cumulus as the core. I would do the same for daac_version to keep them consistent. Unless t